scss/standalone/patterns_split-list.scss

```diff
@@ -0,0 +1,4 @@
+@import '../vanilla';
```

Comment:
We have to figure out how to get rid of those for patterns. 

Patterns by definition depend on other components, we would have to list them all here. There is no point.

This one is likely missing a bunch as well: grid, rule, CTA, buttons, etc…